### PR TITLE
Issue 31: Add support for standalone Heapster option. For HPA.

### PR DESCRIPTION
**Pull request by mwringe**

### CODE REVIEW

File: deployer/run-hawkular.sh [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    openshift admin ca create-server-cert  \
+      --key=$dir/hawkular-metrics.key \
+      --cert=$dir/hawkular-metrics.crt \
+      --hostnames=hawkular-metrics,${hawkular_metrics_hostname} \
+      --signer-cert="$dir/ca.crt" --signer-key="$dir/ca.key" --signer-serial="$dir/ca.serial.txt"
+    cat $dir/hawkular-metrics.key $dir/hawkular-metrics.crt > $dir/hawkular-metrics.pem
+    cp $dir/ca.crt $dir/hawkular-metrics-ca.cert  
+fi
+
+# Use existing or generate new Hawkular Cassandra certificates
```
Comment (jcantrill) [inline deployer/run-hawkular.sh]: Is this a duplicate of the block above?

File: deployer/run-hawkular.sh [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+# Use existing or generate new Hawkular Metrics certificates
```
Comment (jcantrill) [inline deployer/run-hawkular.sh]: This could be made into a nice function to reduce redundency

**Comment by jcantrill** [comment]
@mwringe I made a few comments.  Accept them if they make sense.  Please squash and merge.

File: deployer/run-hawkular.sh (hunk 2 above)
Comment (mwringe) [inline deployer/run-hawkular.sh]: What exactly could be made into a function? Just creating the certificates or something more than that?

**Comment by mwringe** [comment]
Updated and squashed.

**Pull request merged**